sdcm/sct_config.py

```diff
@@ -1578,6 +1578,7 @@ class SCTConfiguration(dict):
     ]
     ami_id_params = ['ami_id_db_scylla', 'ami_id_loader', 'ami_id_monitor', 'ami_id_db_cassandra', 'ami_id_db_oracle']
     aws_supported_regions = ['eu-west-1', 'eu-west-2', 'us-west-2', 'us-east-1', 'eu-north-1', 'eu-central-1']
+    additional_data_for_elastic = {}
```

Comment:
it's weird that the configuration object of SCT is use for storage like this...
no very intuitive, and I don't want people to start assuming they can change things on `params` during the test case....